code/modules/mafia/controller.dm

```diff
@@ -399,6 +404,8 @@
 	QDEL_NULL(town_center_landmark)
 	phase = MAFIA_PHASE_SETUP
 
+	early_start = FALSE
```

Comment:
```suggestion
	early_start = initial(early_start)
```

more explicit that we're returning it to the default value at the end of a game? idk feel free to ignore this it's a boolean anyways